### DISCUSSION TITLE
Alpha 157

### DISCUSSION THREAD

**Discussion by Eugeny** [Eugeny/tabby#4582] [Release Notes]
* added MSYS2 built-in profiles
* fixed saving profiles
* SSH: reworked keyboard-interactive auth

<hr /><em>This discussion was created from the release <a href='https://github.com/Eugeny/tabby/releases/tag/v1.0.157'>Alpha 157</a>.</em>

**Comment by Jonnyan404** [comment]
#4586  Looking forward to fixing this problem as soon as possible.
